index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>learning gits</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+
+      <img
+        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+        alt="web_image"
+        id="image"
+      />
+      <div id="img-caption">
+        Dr. Norman Borlaug, third from the left, trains biologists in Mexico on
+        how to increase wheat yields - part of his life-long war on hunger.
+      </div>
+
+      <div id="tribute-info">
+        <h4>Here's a time line of Dr. Borlaug's life:</h4>
+        <ul>
+          <li><b>1914</b> - Born in Cresco, Iowa</li>
+          <li>
+            <b>1933</b> - Leaves his family's farm to attend the University of
+            Minnesota, thanks to a Depression era program known as the "National
+            Youth Administration"
+          </li>
+          <li>
+            <b>1935</b> - Has to stop school and save up more money. Works in
+            the Civilian Conservation Corps, helping starving Americans. "I saw
+            how food changed them", he said. "All of this left scars on me."
+          </li>
+          <li>
+            <b>1937</b> - Finishes university and takes a job in the US Forestry
+            Service
+          </li>
+          <li>
+            <b>1938</b> - Marries wife of 69 years Margret Gibson. Gets laid off
+            due to budget cuts. Inspired by Elvin Charles Stakman, he returns to
+            school study under Stakman, who teaches him about breeding
+            pest-resistent plants.
+          </li>
+          <li>
+            <b>1941</b> - Tries to enroll in the military after the Pearl Harbor
+            attack, but is rejected. Instead, the military asked his lab to work
+            on waterproof glue, DDT to control malaria, disinfectants, and other
+            applied science.
+          </li>
+          <li>
+            <b>1942</b> - Receives a Ph.D. in Genetics and Plant Pathology
+          </li>
+        </ul>
+      </div>
+      <a
+        href="https://en.wikipedia.org/wiki/Norman_Borlaug"
+        id="tribute-link"
+        target="_blank"
+        rel="noopener"
+        >Read more</a
+      >
+    </main>
+    <script
+      src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"
+      rel="noopener"
+    ></script>
+
+    <!--     
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team -->
```

Comment:
Please do not leave used code as a comment in your codebase. Unused code as comments makes your code not clean.

Reply:
Good job 👍.